recipes/r-saige/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
     - patches/0001-use-shared-libs-in-makevars.patch
 
 build:
-  number: 0
+  number: 3
```

Comment:
please use 1 instead

Reply:
@bgruening sorry, there is a reason why I ended up with 3.

I tried with 1 first but CI complained that build 1 already exists for this version: https://github.com/bioconda/bioconda-recipes/pull/32545/commits/cae74ff031fd60cbb5bd0c9606240f055066f7de

Any idea why there are builds: `1` and `2` in the conda repo, that were not committed in recipe? 
https://anaconda.org/bioconda/r-saige/files

Reply:
Oh! Yes, please look at the bulk branch. @dpryan79 is currently rebuilding packages there. But this also means build 1 or 2 are recent rebuilds and this PR is not needed anymore, left?

Reply:
@bgruening I actually need it, unfortunately I was patching the dependency after the last rebuild so it is not included,

Reply:
I see, please do not patch your code, but release a new version next time.